tests/functional/conversion/test_ether.py

```diff
@@ -5,7 +5,7 @@
 
 from ape import convert
 from ape.exceptions import ConversionError
-from ape_ethereum.converters import ETHER_UNITS
+from ape_ethereum._converters import ETHER_UNITS
```

Comment:
Why this rename?

Reply:
Because of a docs namespace collision, I think I added a comment about it somewhere